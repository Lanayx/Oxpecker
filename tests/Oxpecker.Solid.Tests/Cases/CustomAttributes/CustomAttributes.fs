module Oxpecker.Solid.Tests.Cases.CustomAttributes

open Oxpecker.Solid

[<SolidComponent>]
let Test () =
    body().attr("hello", "world") {
        h1().data("abcd", "efgh")
    }
