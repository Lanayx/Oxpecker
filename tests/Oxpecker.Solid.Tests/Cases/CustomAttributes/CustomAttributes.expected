
export function Test() {
    return <body hello="world">
        <h1 data-abcd="efgh"></h1>
    </body>;
}

